english/froboznia/froboznia.md

```diff
@@ -1,3 +1,10 @@
 You find yourself face to face with the Froboznian police.
 
 "Ouzna gotcha!" they bellow.
+
+You decide to
+
+-> [Surrender](Surrender\Surrender.md)
+-> [Fight Back](FightBack\FightBack.md) 
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.